cmd/operator-sdk/alpha/scorecard/cmd.go

```diff
@@ -15,33 +15,51 @@
 package scorecard
 
 import (
-	log "github.com/sirupsen/logrus"
-	"github.com/spf13/cobra"
-)
+	"fmt"
+	"log"
 
-var (
-	config   string
-	bundle   string
-	selector string
-	listAll  bool
+	scorecard "github.com/operator-framework/operator-sdk/internal/scorecard/alpha"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/labels"
 )
 
 func NewCmd() *cobra.Command {
+	var (
+		config   string
+		bundle   string
+		selector string
+		list     bool
+	)
 	scorecardCmd := &cobra.Command{
 		Use:    "scorecard",
 		Short:  "Runs scorecard",
 		Long:   `Has flags to configure dsl, bundle, and selector.`,
 		Hidden: true,
 		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
I think we want this to be:
```suggestion
		RunE: func(cmd *cobra.Command, args []string) error {
```

Flag parsing and setup errors should be returned so that we print usage. And any errors after that (i.e. starting with `RunTests`) should be `log.Fatal`